anvio/docs/artifacts/functions.md

```diff
@@ -0,0 +1,11 @@
+This artifact **contains information about the functions of the contigs in your %(contigs-db)s.**
+
+It is less a separate file, and more an extension of your %(contigs-db)s that contains this information. 
+
+To get one of these for your %(contigs-db)s, you can either import it (using %(anvi-import-functions)s) or make one yourself by running your contigs against one of two databases available in anvi'o:
+* NCBI [COGs database](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC102395/) -- see %(anvi-run-ncbi-cogs)s for instructions
+* EBI's [Pfam database](https://pfam.xfam.org/) -- see %(anvi-run-pfams)s for instructions
+
+This is used to run %(anvi-analyze-synteny)s. 
+
+You can also use %(anvi-export-functions)s to view the contents of this file through a %(functions-txt)s artifact. 
```

Comment:
Instead of 'view the contents of this file', I would say 'obtain a file containing functional annotation information'  :)